README.md

```diff
@@ -37,6 +37,59 @@ Please kindly check [[Link]](https://github.com/indobenchmark/indonlu/tree/maste
 
 For the submission, first you need to rename your prediction into 'pred.txt', then zip the file.
 
+## Quick Start
+
+### Predict
+_TBD_
+
+### Train
+_TBD_
+
+### Reproduce Result
+
+1. Set the `CUDA_VISIBLE_DEVICES` environment variable first
+    ```
+    export CUDA_VISIBLE_DEVICES=0
+    ```
+2. Then, simply execute the following command to run the training
+    ```
+    make reproduce DATASET=<dataset>
+    ```
+    It will train all of the models for the specified _\<dataset\>_ with default parameter
+3. Check the available datasets in `datasets/` directory
+4. All of the models used are listed in `scripts/config/model/train.yaml` \
+    Feel free to add or comment as you see fit
+5. To use different hyperparameter, create a new file in `scripts/config/hyperparameter/` \
+    Then specify it in the command like this
+    ```
+    make reproduce DATASET=<dataset> HYPERPARAMETER=<hyperparameter_filename_without_the_extension>
+    ```
+6. There are 2 more parameters that can be specified in the command:
+    - EARLY_STOP
+    - BATCH_SIZE
+
+    Use the following command to utilize it
+    ```
+    make reproduce DATASET=<dataset> EARLY_STOP=<early_stop> BATCH_SIZE=<batch_size>
+    ```
+7. There are also a grouping command of specific task for easy access like
+    ```
+    make reproduce_all_1
+    make reproduce_all_2
+    etc
+    ```
+
+##### Non Pretrained No Special Token
```

Comment:
I think we can remove this one, all no special token experiment is just an experimental before. I feel it will make people confuse.